frontend/src/metabase/containers/ItemPicker/ItemPicker.unit.spec.js

```diff
@@ -241,4 +294,134 @@ describe("ItemPicker", () => {
     const list = within(getItemPickerList());
     expect(list.getByText(DASHBOARD.REGULAR_CHILD.name)).toBeInTheDocument();
   });
+
+  describe("preserves order of collections coming from API endpoint", () => {
+    it("[personal, regular, regular 2]", async () => {
+      const collections = [
+        COLLECTION.PERSONAL,
+        COLLECTION.REGULAR,
+        COLLECTION.REGULAR_2,
+      ];
+
+      fetchMock.get("path:/api/collection", collections);
```

Comment:
We setup fetch mocks within the `setup` function, and use mocks defined in `server-mocks`

Reply:
This was a leftover, good catch, removed: 4d444a33f293169f855ace66217f27f6a86c81bf

Reply:
+ 066febb437e5081f5f14a37e007dcc80d2269363